BlazarData/src/main/java/com/hubspot/blazar/data/dao/DependenciesDao.java

```diff
@@ -38,6 +39,15 @@
   @SqlQuery("SELECT * FROM module_depends WHERE moduleId = :moduleId")
   Set<Dependency> getDependencies(@Bind("moduleId") int moduleId);
 
+  @SqlQuery("SELECT gitInfo.* " +
```

Comment:
Yeah this is a big refactor, I think best taken care of in a separate PR given how far it reaches.
We also have to be very careful w/ deployment of that change because the executor isn't deployed at the same time.